src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,53 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationException("User can`t be null!");
+        }
+
+        if (user.getLogin() == null || user.getLogin().length() < 6) {
```

Comment:
You're checking if the login is null or less than 6 characters long. However, the checklist mentions that you should not add anything that is not described in the task requirements. The task does not specify a minimum login length.